### DISCUSSION TITLE
Add more identifiers of other resources

### DISCUSSION THREAD

**Discussion by jaschaurbach** [bookwyrm-social/bookwyrm#2370] [Ideas]
I was thinking: How to make books more accessible? O course, use more identifiers! 

I e.g. read read many magazines and if they are not zines but "regularly" published by a publisher there is a very good chance that it exists in the "Zeitschritendatenbank (ZDB)" (Magazine Database) and the whole series has an ID there.

Example: Queer*Welten, a queer sci-fi and fantasy short stories magazine, has the ID 3027928-8 which gives the permalink https://ld.zdb-services.de/resource/3027928-8. On this page I can see which libraries in Germany do have it and which one do interlending? (of course only if they participate and enter they information at the ZDB - in fact, many libraries do so)

The ZDB-ID can be fetched from open library when present and I am just in the process entering all the information to OL fro things I read. 

**I suggest**: If the ID is present, show the permalink together with all the other links where I can get a book. Make it optional for instance admins to show these links if ID present or do not show it. 

Another one would be: Bookcrossing
Every title and editions gets a title ID (e.g. 6051843 for Cemetery Boys). It expands to https://www.bookcrossing.com/title/6051843/ and you can see how many of them are registered there available to be crossed. 

Caution: The Bookcrossing ID in openlibrary points to a Bookcrossing journal entry for exactly _one_ copy and its journal entries, not to the title and this is wrong (I have not found out yet where to report this for OL).

**I suggest**: for every edition in field be filled out with a ID, make a checkbox to let the information "other edition in same language" be shown or not if e.g. the crossing book is 3rd edition but the book in BookWyrm is second edition but both are English.

WorldCat ID: The worldcat shows libraries where a book is available for lending. (if the library participates). the world cat ID expands to a permanent-URL. The books I have published 10 years ago is not available in print any more. the world cat ID is 928088694. It expands to permalink https://www.worldcat.org/title/928088694 and all libraries which have it are shown. People can get and read it without spending _more_ money.

I suggest: Make it a possibility like I suggested prior for ZDB-IDs on instance level to show the WordlCat link.

**Comment by mouse-reeve** [comment]
Thank you! To summarize, it sounds like you're suggesting adding the following identifiers to books:
 - ZBD
 - Bookcrossing
 - Worldcat
 - DNB (which iirc you mentioned in a different issue)

And also providing links to worldcat when the worldcat id is available.

Since these are actionable items, I'm going to convert this from a discussion to an issue.

**Reply by jaschaurbach** [reply]
I raised a discussion because I was not sure if it should be that much.

The DNB ID would be nice to have if it is used later e.g. for getting covers from the DNB or anything. otherwise it is quite useless.

**Reply by mouse-reeve** [reply]
IDs like that, even if they aren't being used for something concrete, can still be useful in deduplicating books when they are loaded into an instance

**Reply by jaschaurbach** [reply]
Just forgot that (I think I spoke about that as well). I am tired. Sorry.

**Reply by mouse-reeve** [reply]
all good! I really appreciate all your contributions and ideas :)

**Reply by jaschaurbach** [reply]
Thanks. please stop me if it is getting pout of hand :D

**Comment by jaschaurbach** [comment]
I have some more identifiers: 

~~isfdb.org both for books as well as for authors.~~ (done with #2482 and #2501)
Books can be queried by API https://isfdb.org/wiki/index.php/Web_API (if there is an ISBN we could ask the API and look up the ID or the other way around).

The GND identifier for authors (it is an identifier for people, places, etc. important for us would be the ones for people). https://www.dnb.de/EN/Professionell/Standardisierung/GND/gnd_node.html
They can be linked to the entry of this author at the DNB https://d-nb.info/gnd/1077180748 (e.g. this is mine)
